.github/workflows/scripts/windows-container-prep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# abstract the prepare commands for the windows build into a script that can be reused
+# instead of a yaml file
+cd /win
+ln -s /__w/subsurface/subsurface .
+ls -l
+ls -l subsurface/scripts
```

Comment:
hehe - I know all about lines like this to make sure my script is in the right spot... 
and then I keep forgetting to remove them before pushing things to master :-/